cmd/publish/main_test.go

```diff
@@ -84,6 +84,7 @@ func TestPublishCommand(t *testing.T) {
 		Messager:         messagerMock,
 		Printer:          printerMock,
 		PublishCliClient: publishClientMock,
+		FilesExtractor:   &files.FilesExtractor{},
```

Comment:
I see no value for mocking the file extractor in this test, therefore changed to the real implementation :)

Reply:
Hello @royhadad , there is a for mocking `FilesExtractor`. Without it, I can't mock the output of `ExtractFilesConfigurations` function and which is then passed to the `ValidateResources` as arguments. Thus I can check whether the `ValidateResources` function has the right parameters or not using `AssertCalled`

Reply:
why would you care if it was called with the right parameters? that's an implementation detail